electrum_nmc/electrum/names.py

```diff
@@ -681,6 +681,18 @@ def get_domain_records_address(domain, value):
         if value["freenet"] is None:
             del value["freenet"]
 
+    if "ipfs" in value:
+        new_records, value["ipfs"] = get_domain_records_address_dnslink(domain, value["ipfs"])
+        records.extend(new_records)
+        if value["ipfs"] == []:
+            del value["ipfs"]
+    
+    if "ipns" in value:
+        new_records, value["ipns"] = get_domain_records_address_dnslink(domain, value["ipns"])
+        records.extend(new_records)
+        if value["ipns"] == []:
+            del value["ipns"]
+
```

Comment:
I'm not sure I understand what these are for. It looks like they only activate when there's an `ipfs` or `ipns` JSON field... but then it loads the result from DNSLink instead of those fields. Can we just omit these? Presumably they'll get handled by the `txt` case later, right?